core/src/display_object/movie_clip.rs

```diff
@@ -1695,6 +1695,10 @@ impl<'gc> MovieClip<'gc> {
                 .any(|handler| object.has_property(&mut activation, handler.into()))
         }
     }
+
+    pub fn event_trigger(self, context: &mut UpdateContext<'_, 'gc, '_>, event: ClipEvent<'gc>) {
+        self.event_dispatch(context, event);
+    }
```

Comment:
Can we directly call `event_dispatch` instead of adding this function?

Reply:
`event_dispatch` is part of the `TInteractiveObject` trait, so it will work if you import it at the top of loader.rs:
```
use crate::display_object::{...etc, TDisplayObjectContainer};
```

Reply:
Thank you, this worked. 😉